pkg/exchange/coinbase/convert.go

```diff
@@ -12,12 +12,20 @@ import (
 
 func toGlobalOrder(cbOrder *api.Order) types.Order {
 	return types.Order{
+		SubmitOrder: types.SubmitOrder{
+			ClientOrderID: cbOrder.ClientOID,
+			Side:          cbOrder.Side.GlobalSideType(),
```

Comment:
use toGlobalSide() instead of method,

because binding method will import the global types package from the api package.